src/transaction-flow/input/ProfileEditor/ProfileEditor-flow.tsx

```diff
@@ -275,7 +277,13 @@ const ProfileEditor = ({ data = {}, transactions = [], dispatch, onDismiss }: Pr
                     validator={validatorForRecord(field)}
                     validated={isDirtyForRecordAtIndex(index)}
                     error={errorForRecordAtIndex(index, 'key')}
-                    onDelete={() => handleDeleteRecord(field, index)}
+                    onDelete={() => {
+                      if (isEthAddressRecord(field)) {
+                        updateRecordAtIndex(index, { ...field, value: '' })
+                      } else {
```

Comment:
This is unnecessary since the field group is "custom" which means that it will always be a text record.